.github/workflows/release.yml

```
@@ -20,10 +20,8 @@ jobs:
         with:
           node-version: lts/*
           cache: npm
-      - run: >-
-          npm install -D semantic-release
-          semantic-release-plugin-update-version-in-files
-      - run: npx semantic-release --debug
+      - run: npm install --frozen-lockfile
```

This is a yarn flag - you want `npm ci`

```suggestion
      - run: npm ci
```

---

Thanks!